developer-docs-site/docs/cli-tools/aptos-cli-tool/automated-install-aptos-cli.md

```diff
@@ -48,3 +54,24 @@ iwr "https://aptos.dev/scripts/install_cli.py" -useb | Select-Object -ExpandProp
 ```
 
 </details>
+
+## Update
+
+To trigger automatic updates, run: `aptos update` and see output indicating success:
```

Comment:
I wouldn't say it's triggering an automatic update, that might imply that the CLI will update itself without you doing anything. If you run `aptos update` you're telling the CLI to update itself, so it's really a manual invocation.

So instead just "To update the CLI, run..."

Reply:
Thanks, Daniel! I believe this is now done.